style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: poppins, serif;
+}
+
+main {
+  width: 375px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  min-width: 375px;
+  padding: 24px  24px;
+}
+
+section {
+  background-image: url(./images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 24px 24px;
+}
+
+.first-heading {
+  margin-top: 60px;
+  font-size: 2rem;
+  font-weight: bold;
+  color: #091e42;

```

Comment:
- [ ] kindly use this color` [#6070FF]` for the main heading/title(i am ,,,,)

- [ ] use this color `(#6070FF)` for your logo so it matches the design